package.json

```diff
@@ -23,9 +23,9 @@
     "lock": "npm run lint:versions -s && npm shrinkwrap --dev && sed -i '' 's/https:/http:/g' npm-shrinkwrap.json"
   },
   "dependencies": {
-    "jquery": "^2.2.4",
-    "uikit": "3.0.0-beta.23",
-    "throttle-debounce": "^1.0.1"
+    "jquery": "2.2.4",
+    "uikit": "3.0.0-beta.27",
+    "throttle-debounce": "1.0.1"
```

Comment:
Wouldn't it be better to allow minor (or at least patch) releases, rather than having to update the boilerplate manually? The `package-lock.json` file should let the individual projects stick to their specific version.